planner/core/rule_inject_extra_projection.go

```diff
@@ -121,11 +121,10 @@ func injectProjBelowAgg(aggPlan PhysicalPlan, aggFuncs []*aggregation.AggFuncDes
 	}
 
 	child := aggPlan.Children()[0]
-	prop := aggPlan.GetChildReqProps(0).Clone()
 	proj := PhysicalProjection{
 		Exprs:                projExprs,
 		AvoidColumnEvaluator: false,
-	}.Init(aggPlan.context(), child.statsInfo().ScaleByExpectCnt(prop.ExpectedCnt), prop)
+	}.Init(aggPlan.context(), child.statsInfo()) // give a nil property since physical optimization has finished.
```

Comment:
Move this comment to line 124.
We pass a nil property to `Init` since PhysicalOptimization phase has finished.

Reply:
comment has been updated.